lib/plugins/terser.js

```diff
@@ -18,7 +18,9 @@ const applyOptionsCallback = require('../utils/apply-options-callback');
  */
 module.exports = function(webpackConfig) {
     const terserPluginOptions = {
-        sourceMap: webpackConfig.useSourceMaps
+        sourceMap: webpackConfig.useSourceMaps,
+        cache: true,
+        parallel: true,
```

Comment:
Is it normal to end the last line with a comma?

Reply:
Yes this is allowed in JavaScript (like in TypeScript, Python, PHP...).

However I'm not sure there is a specifing style/linting rule on Encore.


See https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Virgules_finales for JavaScript